client/src/components/Home/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import swal from 'sweetalert';
+import { Link } from 'react-router-dom';
+import { Grid, Box, Typography, Paper, IconButton } from '@material-ui/core';
+import { AddCircleOutline } from '@material-ui/icons';
+import LoaderProgress from '../../common-components/LoaderProgress';
+import Water from '../../assets/water.svg';
+import Communication from '../../assets/communication.svg';
+import Electricity from '../../assets/electricity.svg';
+import Internet from '../../assets/internet.svg';
+import Styles from './style';
+
+export default () => {
+  const classes = Styles();
+  const [userBillType, setUserBillType] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [displayBlock, setIsDisplayBlock] = useState(true);
+
+  useEffect(() => {
+    axios
+      .get('/api/home')
+      .then((result) => {
+        setUserBillType(result.data.Result);
+      })
+      .catch((err) => {
+        if (err.response.data) {
+          swal('Error', err.response.data.message, 'error');
+        }
+        setIsLoading(false);
+      });
+  }, []);
+
+  const BillCard = (index, type, imgSrc) => (
+    <Link to={`/bill/${type}`}>
+      <Paper key={index} className={classes.root} elevation={3}>
+        <img src={imgSrc} className={classes.imageType} />
+      </Paper>
+    </Link>
+  );
+
+  const buildBillType = (types) => {
+    if (types.length === 0) {
+      return [];
+    }
+    return types.map((type, index) => {
+      if (types.length - 1 === index && isLoading) {
+        setIsLoading(false);
+      }
+      switch (type.type.toLocaleLowerCase()) {
+        case 'electricity':
+          return BillCard(index, 'electricity', Electricity);
+        case 'water':
+          return BillCard(index, 'water', Water);
+        case 'internet':
+          return BillCard(index, 'internet', Internet);
+        case 'communication':
+          return BillCard(index, 'communication', Communication);
+        default:
+          return (
+            <Paper key={index} className={classes.root} elevation={3}>
+              <Typography variant="h5">Unknown</Typography>
+            </Paper>
+          );
+      }
+    });
+  };
+
+  const displayStatus = isLoading && !displayBlock ? 'none' : 'block';
+
+  return (
+    <Box component="div" p={3} width={1}>
+      <LoaderProgress isLoading={isLoading} />
+      <Box component="div" display={displayStatus} width={1}>
+        <Grid container item sx={12} justify="center">
+          <Grid item container xs={12}>
+            <Typography variant="h4" color="textPrimary" align="left">
+              Billbase
+            </Typography>
```

Comment:
we want a header component to render the title of the page and menu Icon instead of repeating the same code in all pages.